addons/website_slides/views/slide_channel_tag_views.xml

```diff
@@ -67,9 +67,10 @@
                     <div class="oe_title">
                         <label for="name" string="Course Group Name"/>
                         <h1><field name="name" default_focus="1" placeholder="e.g. Your Level"/></h1>
-                        <label for="is_published" string="Menu Entry"/>
-                        <field name="is_published"/><br/>
                     </div>
+                    <group>
+                        <field name="is_published" string="Menu Entry"/>
+                    </group>
```

Comment:
I don't think there is a need for a new `group` tag, this could probably be set in the `group` tag below, just above the `tag_ids` field :slightly_smiling_face:

Reply:
It's not bad advice though, and it's also possible that we'd want to reshape the part above as well, in master.